src/synthesizer/components/component.py

```diff
@@ -233,6 +233,7 @@ def get_spectra(
         tau_v=None,
         fesc=None,
         mask=None,
+        shift=False,
```

Comment:
As mentioned `vel_shift` or even `do_vel_shift` might be a better more clear name.